### DISCUSSION TITLE
Feature request: miniUEFI support for edk2-firmware-tegra

### DISCUSSION THREAD

**Discussion by marioal9806** [OE4T/meta-tegra#1753] [Ideas]
Nvidia added support for a [miniUEFI configuration on JP6](https://docs.nvidia.com/jetson/archives/r36.4/DeveloperGuide/SD/Bootloader/UEFI.html#miniuefi-support). Will that UEFI configuration be supported on the `edk2-firmware-tegra` recipe? Is that in the plans for meta-tegra?

**Comment by madisongh** [comment]
I've got changes queued up in #1758 for this, if you'd like to give them a try.

**Reply by marioal9806** [reply]
Thanks @madisongh! I'll try them out!
